flake8.cfg

```diff
@@ -0,0 +1,17 @@
+[flake8]
+exclude = ./docs/*,*/migrations/*
+ignore = Q000,D100,D104,D106,D200,D401,D402
+max-line-length = 100
+
+# Flake8-quotes extension codes
```

Comment:
Can we remove these commented lines?

Reply:
Fine with me.  cut-n-paste from pulp file.  We should open an issue against that project too then.